### PR TITLE
Add license snippet to headers, mention all contributors in debian/copyright

### PR DESCRIPTION
**Pull request by bluca**
Contributors details and dates are taken from git log.

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -4,6 +4,25 @@ Source: https://bues.ch/h/razercfg
...
+           2014 perfect7gentleman <perfect007gentleman@gmail.com>
+           2014 polyzen <polycitizen@gmail.com>
+           2014 Ryan Mulcahy <mulchy81@gmail.com>
+           2014 Tony Crisci <tony@dubstepdish.com>
+           2014 Torstein Husebø <torstein@huseboe.net>
+           2015 Corentin Jabot <corentinjabot@gmail.com>
+           2015 Greg Larmore <larmoreg@gmail.com>
+           2015 Konrad Zemek <konrad.zemek@gmail.com>
+           2016 Jan Stümmel <info@janstuemmel.de>
+           2016 Luca Weiss <z3ntu@users.noreply.github.com>
```
Comment (z3ntu) [inline debian/copyright]: You can update my email address to `luca@z3ntu.xyz`

Comment (bluca) [reply debian/copyright]: Done, thanks

**Comment by mbuesch** [comment]
There are no copyright notices in the headers, because I don't think headers are copyrightable on their own.
So do we really need this? I'd rather like to keep the headers as-is.

**Comment by bluca** [comment]
Sure, no problem

**Pull request merged**